pkg/testinteraction/common.go

```diff
@@ -0,0 +1,74 @@
+package testinteraction
```

Comment:
I would have expected to see this in testcommon, but don't have any objection if this is where you want it.